### PR TITLE
Update the readme, ensure the "simple" setup path works

### PR DESCRIPTION
**Pull request by gemmaleigh**
Update the readme:
* remove yarn, using npm install works as expected and shortens the installation steps.
* add detail for configuring this application

Amend server.js:
* ensure that if the default stylesheet is used, paths to image assets for the template and toolkit are not broken

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,21 +11,55 @@ It's a Node & ExpressJS app, with Nunjucks for templating.
...
+
+Static content is served from the public directory:
+
+```javascript
+app.use('/public', express.static(path.join(__dirname, '/public')))
+```
+
+## Using GOV.UK frontend
+
+## Default
```
Comment (robinwhittleton) [inline README.md]: These are subsections so should be `###` right?

File: README.md [hunk 2]
```diff
@@ -11,21 +11,55 @@ It's a Node & ExpressJS app, with Nunjucks for templating.
...
+For the default compiled stylesheet only - this application has been set to serve the govuk_frontend_alpha toolkit and template assets from the node_modules directory:
+
+In `server.js`:
+
+```javascript
+app.use('/images/toolkit', express.static(path.join(__dirname, '/node_modules/govuk_frontend_alpha/assets/images/toolkit/')))
+app.use('/images/template', express.static(path.join(__dirname, '/node_modules/govuk_frontend_alpha/assets/images/template/')))
+```
+
+## Custom - replacing the default stylesheet
```
Comment (robinwhittleton) [inline README.md]: Same here, `###`

File: README.md [hunk 3]
```diff
@@ -11,21 +11,55 @@ It's a Node & ExpressJS app, with Nunjucks for templating.
...
+To serve the govuk_frontend_alpha assets from the node_modules directory, in `server.js` use:
+
+```javascript
+app.use('/public', express.static(path.join(__dirname, '/node_modules/govuk_frontend_alpha/assets/')))
+```
+
+To ensure the path to the image assets is correct, `$path` must be defined at the top of `application.scss`, find this in:
+
+```
+assets/scss/application.scss`
```
Comment (robinwhittleton) [inline README.md]: There’s an extra ` on the end of this path. Actually, it should probably just be inline with the above para like other paths in the file?

**Comment by gemmaleigh** [comment]
@robinwhittleton, thanks! i've made those changes.

**Pull request merged**